### DISCUSSION TITLE
Which module is preferred for Karpenter installation?

### DISCUSSION THREAD

**Discussion by asychev** [aws-ia/terraform-aws-eks-blueprints-addons#351] [Q&A]
This module supports Karpenter provisioning, same as [terraform-aws-eks](https://github.com/terraform-aws-modules/terraform-aws-eks) (via [submodule](https://github.com/terraform-aws-modules/terraform-aws-eks/tree/master/modules/karpenter)).
Are there particular reasons to have this duplication? What is the preferred way?

**Comment by bryantbiggs** [comment]
The EKS module does not provision Karpenter - it only provides the necessary AWS infrastructure resources for Karpenter

**Reply by asychev** [reply]
Yes, but they still overlap. Do I understand correctly this module not yet support v20.x EKS clusters created with `authentication_mode = "API"`?

**Reply by bryantbiggs** [reply]
>  Do I understand correctly this module not yet support v20.x EKS clusters created with authentication_mode = "API"

Correct - that would be something incorporated into v2.0 of this module
